pegasus/sites.v3/code.org/public/professional-learning/international.haml

```diff
@@ -0,0 +1,65 @@
+---
+title: International Professional Resources
```

Comment:
does the title need to be translated?

Reply:
Ideally yes, but now that you mention it I don't think I've seen any titles translated on Pegasus pages 🤔 Do you know if this is possible?

Reply:
I guess not :/ that's a bit of a bummer

Reply:
I did some more digging and found out how to do it! 🎉 ([here's an example](https://github.com/code-dot-org/code-dot-org/blob/e7c0b75a16c73ba132d3b616913abcd894401aee/pegasus/sites.v3/code.org/public/index.haml#L13-L14) on the homepage) 

I'll tag you on the commit shortly.